static/src/javascripts/bootstraps/enhanced/youtube.js

```diff
@@ -0,0 +1,92 @@
+define([
+    'fastdom',
+    'common/modules/video/youtube-player',
+    'common/modules/video/youtube-tracking',
+    'common/utils/$'
+], function (
+    fastdom,
+    youtubePlayer,
+    tracking,
+    $
+) {
+
+    var STATES = {
+        'ENDED': _onPlayerEnded,
+        'PLAYING': _onPlayerPlaying,
+        'PAUSED': _onPlayerPaused
+    };
+
+    var progressTracker = {};
+    var players = {};
+
+    function init() {
+
+        fastdom.read(function () {
+                        $('.atom--media--youtube').each(function (el) {
+                            var atomId = el.getAttribute('data-media-atom-id');
+                            var videoId = el.firstElementChild.id;
+                            tracking.init(atomId);
```

Comment:
@akash1810 This is because hosted uses youtube-id's whereas we use atom-id's.  Hence the generic video-id identifier.  I agreed to make the hosted switch to atom-id's in a separate PR.  I've updated the videoID identifier in youtube-player to be named youtube-id to be more explicit.